topdown/encoding.go

```diff
@@ -158,6 +158,29 @@ func builtinURLQueryEncodeObject(a ast.Value) (ast.Value, error) {
 	return ast.String(query.Encode()), nil
 }
 
+func builtinURLQueryDecodeObject(bctx BuiltinContext, operands []*ast.Term, iter func(*ast.Term) error) error {
+	query, err := builtins.StringOperand(operands[0].Value, 1)
+	if err != nil {
+		return err
+	}
+
+	queryParams, err := url.ParseQuery(string(query))
```

Comment:
I'm using ParseQuery here, but if it is better to also include the "?" in the query string we could use https://golang.org/pkg/net/url/#URL.Query instead

Reply:
I think this version is fine.